packages/server-side-render/src/server-side-render.js

```diff
@@ -69,7 +69,22 @@ function DefaultErrorResponsePlaceholder( { response, className } ) {
 	return <Placeholder className={ className }>{ errorMessage }</Placeholder>;
 }
 
-function DefaultLoadingResponsePlaceholder( { children, showLoader } ) {
+function DefaultLoadingResponsePlaceholder( { children, isLoading } ) {
+	const [ showLoader, setShowLoader ] = useState( false );
+
+	useEffect( () => {
+		if ( ! isLoading ) {
+			setShowLoader( false );
```

Comment:
Note: This is just a safeguard for the future. Currently, the component will unmount when `isLoading` settles on `false`.